extensions/blocks/podcast-player/style.scss

```diff
@@ -160,19 +159,19 @@ $jetpack-podcast-player-background: $white;
 
 	.jetpack-podcast-player__audio-player {
 		height: 40px; // mirroring .mejs-container
-		margin-bottom: $block-grid-gutter;
+		margin-bottom: $gutter-l;
 	}
 
 	.jetpack-podcast-player--audio-player-loading {
 		height: 10px; // mirroring .mejs-time-total
 		background: $jetpack-podcast-player-secondary;
-		margin: 15px $block-grid-gutter; // simulating spacing of .mejs-container
+		margin: 15px $gutter-l; // simulating spacing of .mejs-container
```

Comment:
Use `$gutter-m` for `15px` here :)

Reply:
Replaced in https://github.com/Automattic/jetpack/pull/15512/commits/f6a18391d3338752b1736e77ff62f92211583a51